### PR TITLE
[RELEASE] Fix preset loadable

### PR DESCRIPTION
**Pull request by antonrohr**

### CODE REVIEW

File: src/OrbitGl/App.cpp [hunk 1]
```diff
@@ -2087,7 +2087,6 @@ void OrbitApp::RefreshFrameTracks() {
     GetMutableTimeGraph()->RemoveFrameTrack(function_id);
     AddFrameTrackTimers(function_id);
   }
-  GetMutableTimeGraph()->GetTrackManager()->RequestTrackSorting();
```
Comment (reichlfl) [inline src/OrbitGl/App.cpp:2090]: Is this in here by accident? This will break the change Pablo & I did yesterday.

**Review by reichlfl — CHANGES_REQUESTED** [review]

**Comment by antonrohr** [comment]
Sorry, there was an accident. will fix

File: src/OrbitGl/App.cpp (hunk 1 above)
Comment (antonrohr) [reply src/OrbitGl/App.cpp:2090]: yes, sorry

**Comment by antonrohr** [comment]
fixed

**Review by dimitry- — APPROVED** [review]

**Review by reichlfl — APPROVED** [review]
LGTM, thanks!

**Review by beckerhe — APPROVED** [review]
LGTM. Thanks Anton

**Pull request merged**